Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -1606,7 +1606,8 @@ private KeyCode MapKey (WindowsConsole.ConsoleKeyInfoEx keyInfoEx)
                 }
             }
 
-            return (KeyCode)keyInfo.KeyChar;
+            // Return the Key (not KeyChar!)
```

Comment:
Note (not about this change specifically but the code above it).

Lines 1585 through 1594 are redundant with the block immediately following on line 1595 and can just be removed, unless this is supposed to be doing something else.

Reply:
Nope.  1589 checks for `&&` and 1595 checks for `||`.